openqa-investigate

```diff
@@ -242,6 +242,8 @@ main() {
         client_call=(openqa-cli "${client_args[@]}")
         if [[ -n "$client_prefix" ]]; then
             client_call=("$client_prefix" "${client_call[@]}")
+        else
+            client_call=("runcli" "${client_call[@]}")
```

Comment:
can this be combined into something like

```suggestion
        client_prefix="${client_prefix:-runcli}"
        client_call=("$client_prefix" "${client_call[@]}")
```

Reply:
done